<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportProgress</class>
 <widget class="QDialog" name="ExportProgress">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Exporting...</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>583</width>
        <height>401</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTextEdit" name="lbExport">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QProgressBar" name="pgExport">
       <property name="value">
        <number>0</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
       <property name="textDirection">
        <enum>QProgressBar::BottomToTop</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btEnd">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
